src/EnergyPlus/WaterCoils.cc

```diff
@@ -2622,8 +2638,19 @@ namespace WaterCoils {
                     WaterCoil(CoilNum).DesInletAirHumRat = TempSize; // coil report
                     WaterCoil(CoilNum).InletAirHumRat = TempSize;
                     TempSize = AutoSize; // these data are initially 0, set to autosize to receive a result from RequestSizing
-                    RequestSizing(state, CompType, CompName, HeatingAirflowUASizing, SizingString, TempSize, bPRINT, RoutineName);
+                    //TODO: RequestSizing(state, CompType, CompName, HeatingAirflowUASizing, SizingString, TempSize, bPRINT, RoutineName);
                     // WaterCoil( CoilNum ).InletAirMassFlowRate = TempSize;
+                    EnergyPlus::HeatingAirflowUASizer sizer;
+                    EnergyPlus::HeatingAirflowUASizerFlags flags;
+                    flags.curTermUnitSizingNum = DataSizing::CurTermUnitSizingNum;
+                    flags.curZoneEqNum = DataSizing::CurZoneEqNum;
+                    sizer.zoneSizingInput = DataSizing::ZoneSizingInput;
+                    flags.termUnitSingDuct = TermUnitSingDuct;
+                    flags.termUnitPIU = TermUnitPIU;
+                    flags.termUnitIU = TermUnitIU;
```

Comment:
If these flags were moved to baseFlags then these wouldn't need to be set for each sizing type. Similar for the sizer.zoneSizingInput field.